index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="refresh" content="0; url=https://docs.docker.com" />
+  <title>Docker docs</title>
+</head>
+<body></body>
```

Comment:
Perhaps this body should also mention that we're redirecting; similar to the 404 page, and provide a link if redirect doesn't kick in (e.g., if you're not being automatically redirected, click here ..)